internal/k8s/leader_elector.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 // LeaderElectorGetter is an interface for initialize and check elected leader
+//
 //go:generate mockgen -destination=mocks/leader_elector_getter_mocks.go -package=mocks github.com/dell/csm-metrics-powerscale/internal/k8s LeaderElectorGetter
```

Comment:
@baoy1 Curious why having those empty comments lines?

Reply:
Looks like it is new format rule in go 1.19, otherwise the format check will fail.